src/Analyzers/VisualBasic/Analyzers/ConvertTypeofToNameof/VisualBasicConvertTypeofToNameofDiagnosticAnalyzer.vb

```diff
@@ -0,0 +1,21 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.ConvertTypeOfToNameOf
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.ConvertTypeOfToNameOf
+    <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
+    Friend NotInheritable Class VisualBasicConvertTypeOfToNameOfDiagnosticAnalyzer
+        Inherits AbstractConvertTypeOfToNameOfDiagnosticAnalyzer
+
+        Protected Overrides Function IsValidTypeofAction(context As OperationAnalysisContext) As Boolean
```

Comment:
`NameOf` was not added to Visual Basic till version `14`. We should do a check here to see if the version of VB in use is `14` or higher